docs/how-to/h-deploy-aks.md

```diff
@@ -0,0 +1,235 @@
+Verify that it is correctly installed running the command below:
+
+```shell
+az --version
+```
+
+which should show the following output:
+
+```shell
+azure-cli                         2.65.0
+
+core                              2.65.0
+telemetry                          1.1.0
+
+Dependencies:
+msal                              1.31.0
+azure-mgmt-resource               23.1.1
+
+Python location '/opt/az/bin/python3'
+Extensions directory '/home/deusebio/.azure/cliextensions'
+
+Python (Linux) 3.11.8 (main, Sep 25 2024, 11:33:44) [GCC 11.4.0]
+
+Legal docs and information: aka.ms/AzureCliLegal
+
+
+Your CLI is up-to-date.
+```
+
+## Create AKS cluster
+
+Login to your Azure account:
+```shell
+az login
+```
+
+This following examples in this guide will use the single server AKS in location `eastus` - feel free to change this for your own deployment.
+
+Create a new [Azure Resource Group](https://learn.microsoft.com/en-us/cli/azure/manage-azure-groups-azure-cli):
+
+```shell
+az group create --name aks --location eastus
+```
+
+Export the deployment name for further use:
+```shell
+export JUJU_NAME=aks-$USER-$RANDOM
+```
+
+Bootstrap AKS with the following command (increase nodes count/size if necessary):
+```shell
+az aks create -g aks -n ${JUJU_NAME} --enable-managed-identity --node-count 1 --node-vm-size=Standard_D4s_v4 --generate-ssh-keys
+```
+
+Sample output:
+```yaml
+{
+  "aadProfile": null,
+  "addonProfiles": null,
+  "agentPoolProfiles": [
+    {
+      "availabilityZones": null,
+      "capacityReservationGroupId": null,
+      "count": 1,
+      "creationData": null,
+      "currentOrchestratorVersion": "1.28.9",
+      "enableAutoScaling": false,
+      "enableEncryptionAtHost": false,
+      "enableFips": false,
+      "enableNodePublicIp": false,
+...
+```
+
+Dump newly bootstrapped AKS credentials:
+```shell
+az aks get-credentials --resource-group aks --name ${JUJU_NAME} --context aks
+```
+
+Sample output:
+```shell
+...
+Merged "aks" as current context in ~/.kube/config
+```
+
+You can verify that the cluster and your client `kubectl` CLI is correctly configured by running a simple command, such as:
+
+```shell
+kubectl get pod -A
+```
+
+which should provide the list of the pod services running. 
+
+## Bootstrap Juju controller on AKS
+
+Bootstrap Juju controller:
+```shell
+juju bootstrap aks <CONTROLLER_NAME>
+```
+Sample output:
+```shell
+Creating Juju controller "aks" on aks/eastus
+Bootstrap to Kubernetes cluster identified as azure/eastus
+Creating k8s resources for controller "controller-aks"
+Downloading images
+Starting controller pod
+Bootstrap agent now started
+Contacting Juju controller at 20.231.233.33 to verify accessibility...
+
+Bootstrap complete, controller "aks" is now available in namespace "controller-aks"
+
+Now you can run
+	juju add-model <model-name>
+to create a new model to deploy k8s workloads.
+```
+
+## Deploy Charms
+
+Create a new Juju model, if needed:
+```shell
+juju add-model <MODEL_NAME>
```

Comment:
provide the user with an actual model name. As typed, this command will fail

Reply:
yes, that's expected, as this is not a tutorial but an how-to